package br.com.senai.teste.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity 
@Table (name = "aluno")
public class AlunoModel {
   
    @Id 
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    private int id;

    private String nome, email;

    public AlunoModel() {
}

    public AlunoModel(String nome, String email) {
        this.nome = nome;
        this.email = email;
}

public int getId() {
    return id;
}

public String getNome() {
    return nome;
}

public void setNome(String nome) {
    this.nome = nome;
}

public String getEmail() {
    return email;
}

public void setEmail(String email) {
    this.email = email;
}
}
